dags/openshift_nightlies/config/benchmarks/control-plane-custom.json

```diff
@@ -0,0 +1,37 @@
+{
+    "benchmarks": [
+        {
+            "name": "scale-27",
+            "workload": "scale-perf",
+            "command": "./run_scale_fromgit.sh",
+            "env": {
+                "SCALE": "27",
+                "METADATA_COLLECTION": "true",
+                "WORKLOAD_NODE_ROLE": "workload"
+            }
+        },
+        {
+            "name": "node-density",
+            "workload": "kube-burner",
+            "custom_cmd": "kube-burner ocp node-density --uuid=${UUID} --pods-per-node=245 --timeout=2h --pod-ready-threshold=5s --es-server=${ES_SERVER} --es-index=${ES_INDEX}"
```

Comment:
The variables UUID, ES_SERVER and ES_INDEX are already automatically injected by airflow